openmeter/billing/service/invoicestate.go

```diff
@@ -28,6 +28,18 @@ var invoiceStateMachineCache = sync.Pool{
 	},
 }
 
+var gatheringInvoiceStatusDetails = billing.InvoiceStatusDetails{
+	Immutable: false,
+	// The invoicable state is calculated in the services recalculateGatheringInvoice for now, as the
+	// line data is available there. On the long run we need to cache this information.
+	//
+	// For now, as a safety measure we lie here, as the recalculation will be performed either ways
```

Comment:
is this a placeholder basically that will be updated?  If yes how about doing something like:

`var gatheringInvoiceStatusDetails = billing.NewInvoiceStatusDetails()` or
`var gatheringInvoiceStatusDetails = billing.NewEmptyInvoiceStatusDetails()`

I'm not a fan of word lie. It feels it would fake a status while it's just not calculated yet.